text/0048-schema-annotations.md

```diff
@@ -122,6 +123,92 @@ The same fact also applies to entity attribute declarations.
 This proposal reserves the `annotations` key at the top level of each of those constructs, which contains an Object, containing each annotation key as an Object key, associated with the annotation string.
 The only oddness here is Common Types, whose toplevel is a regular type. While this should still be backwards compatible, it will look a little odd to have annotations in some types and not in others.
 
+A corresponding JSON schema for the above example is as follows.
+```JSON
+{
+    "": {
+    "annotations": {
+        "doc": "this is the namespace"
+    },
+    "commonTypes": {
+        "Task": {
+        "annotations": {
```

Comment:
I assume this matches how we do annotations in the EST?

Reply:
We only allow annotations on policy level in ESTs. So they will on the same level as, e.g., `"effect"`.